src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -230,7 +230,7 @@ public void CollectionObjectsPropertiesNamingValidation()
         public void BodyTopLevelPropertiesValidation()
         {
             var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "body-top-level-properties.json"));
-            messages.AssertOnlyValidationMessage(typeof(BodyTopLevelProperties), 2);
+            messages.AssertOnlyValidationMessage(typeof(BodyTopLevelProperties), 1);
```

Comment:
did the count go down because we're no flagging the inline body parameter issue? If so, I'm ok, because we'd be violating another rule that requests not to have inline body params defined, but we may want to remove that from the spec used for test (as it may confuse us later). It'd be nice to add a comment in the rule itself, that inline body parameters are not being flagged, because we're flagging them as not permitted by another rule.